bin/templates/cordova/lib/builders/GradleBuilder.js

```diff
@@ -93,10 +102,24 @@ GradleBuilder.prototype.prepBuildFiles = function() {
     // Update dependencies within build.gradle.
     var buildGradle = fs.readFileSync(path.join(this.root, 'build.gradle'), 'utf8');
     var depsList = '';
+    var root = this.root;
+    var insertExclude = function(p) {
+          var gradlePath = path.join(root, p, 'build.gradle');
+          var projectGradleFile = fs.readFileSync(gradlePath, 'utf-8');
+          if(projectGradleFile.indexOf('CordovaLib') != -1) {
```

Comment:
What if there is a reference to MyCordovaLib or CordovaLib appears in a comment?
Will it be safe to add the exclude if there is not really a reference to CordovaLib?